### PR TITLE
:broom: Accommodate to knative-sandbox/reconciler-test#294

### PR DESCRIPTION
**Pull request by cardil**
# Changes

- :broom: Accommodate to knative-sandbox/reconciler-test#294

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -134,9 +134,15 @@ require (
...
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	// FIXME: remove this pin when knative/eventing#6451 is merged
+	knative.dev/eventing => github.com/cardil/knative-eventing v0.11.1-0.20220718191420-28d8469b82bf
+	// FIXME: remove this pin when knative-sandbox/reconciler-test#294 is merged
+	knative.dev/reconciler-test => github.com/cardil/knative-reconciler-test v0.0.0-20220720101506-5051d7685fba
+)
```
[the remark this answers is not in the document]
Comment (cardil) [reply go.mod]: Fixed in bb488684db44d78517b9b52ff2ceda55efcaff0e

/unhold

**Review by gab-satchi — APPROVED** [review]
/lgtm

Thanks for the changes

**Pull request merged**